admin/admin-domain-service/ballerina/apiCategoryImpl.bal

```diff
@@ -46,8 +46,10 @@ isolated function getAllCategoryList(commons:Organization org) returns APICatego
             foreach APICategory apiCategory in categories {
                 //TODO:(Sampath) need to properly retrieve attached api count per category 
```

Comment:
let's remove todo

Reply:
Fixed in https://github.com/wso2/apk/pull/1165/commits/13704e6c0a3e4cbb81d7c46e36a0395e2a813b6d